resources/smcp-templates/v2.0/base

```diff
@@ -95,4 +94,6 @@ spec:
           cpu: 10m
           memory: 128Mi
         limits:
+    ior:
```

Comment:
Does it make sense to move this under `pilot`?  Also, since we're moving it, we might see if we can come up with a more descriptive name.

Also, we need to add a field into smcp v2 to enable/disable this feature.

Reply:
@rcernich Have we decided on a name yet? No objections in moving it under `pilot`.
Where is this smcp v2 stuff?

Reply:
@rcernich done. Called it `routeManager`